src/Tribe/Service_Providers/Context.php

```diff
@@ -351,7 +383,8 @@ static function ( $data ) {
 								$date_k = 'tribe-bar-date';
 
 								return is_array( $data ) && isset( $data[ $date_k ] )
-									? add_query_arg( [ $date_k => $data[ $date_k ] ], tribe_get_request_var( 'url', home_url() ) )
+									? add_query_arg( [ $date_k => $data[ $date_k ] ],
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 947e8b985a048c7d6d09063966dd239b</sub>